src/envoy/http/jwt_auth/token_extractor_test.cc

```diff
@@ -154,6 +154,25 @@ TEST_F(JwtTokenExtractorTest, TestCustomHeaderToken) {
   EXPECT_FALSE(headers.get(LowerCaseString("token-header")));
 }
 
+TEST_F(JwtTokenExtractorTest, TestCustomHeaderTokenWithPrefix) {
+  auto headers = TestHeaderMapImpl{{"token-header", "istio jwt_token"}};
+  std::vector<std::unique_ptr<JwtTokenExtractor::Token>> tokens;
+  extractor_->Extract(headers, &tokens);
+  EXPECT_EQ(tokens.size(), 1);
+
+  EXPECT_EQ(tokens[0]->token(), "jwt_token");
+
+  EXPECT_FALSE(tokens[0]->IsIssuerAllowed("issuer1"));
```

Comment:
Does any of these EXPECT have something to do with your changes? I'm curious why they can have different values (i.e, where are these issuers come from? The token is just a fake one, no?)

Reply:
not really, merged to it to [TestCustomHeaderToken](https://github.com/istio/proxy/blob/release-1.1/src/envoy/http/jwt_auth/token_extractor_test.cc#L138), those issues are configured from [kExampleConfig](https://github.com/istio/proxy/blob/release-1.1/src/envoy/http/jwt_auth/token_extractor_test.cc#L31)